### ISSUE TITLE
Does work the page.Screenshot() function with MaskColor?

### ISSUE THREAD

**Issue by Lonka** [playwright-community/playwright-go#388]
Hello, 

I try to screenshot the page and mask the element by different color(#000000), but always mask the default color(#FF00FF).
```go
page.Screenshot(playwright.PageScreenshotOptions{
  Path: playwright.String(screenshotPath),
  Mask: []playwright.Locator{
	  sensElem,
  },
  MaskColor: playwright.String("#000000"),
})
```

Are there any setting I am missing? 

Also, I would like to know dose this MaskColor code could set the transparency value like #80000000 (50%) or rgba(0,0,0, 50)?

thank you.

**Comment by canstand** [comment]
fixed with #389, pls try again.

**Comment by Lonka** [comment]
Hi,

It's perfect for me. I also tried set the `rgba(0,0,0,0.5)` on the MaskColor property. As expected, the elements is masked as transparent.

thank you
